vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -371,6 +371,11 @@ func (s *GenericAPIServer) InstallLegacyAPIGroup(apiPrefix string, apiGroupInfo
 	return nil
 }
 
+func (s *GenericAPIServer) RemoveOpenAPIData() {
```

Comment:
this needs docs. Why do we need this? I remember we found out at KubeCon, but I already forgot.

Reply:
If we do not nil out existing openapi fields for the head of the delegation chain, adding the openapi aggregation controller won't work properly / at all